layouts/_default/content.html

```diff
@@ -1,7 +1,24 @@
 <div class="td-content">
   <h1>{{ .Title }}</h1>
   {{ with .Params.description }}<div class="lead">{{ . | markdownify }}</div>{{ end }}
-  {{ .Content }}
+  {{ $out := .Content}}
+  {{ if not ( in (slice "README.md" "_index.md") .File.LogicalName ) }}
```

Comment:
Will this exclude hardcoded links to other non-Knative Repo's README.md files? Im not sure we still have or need to accommodate those links that point out to another Repo's README file today but maybe we should avoid changing those URLs if or when someone does add on in the future?

Reply:
This is determining if *this file's* name is "README.md" or "_index.md". But I've been adding "index.md" files, so I probably need to update this.

Thanks for asking!